src/coreclr/_build-commons.sh

```diff
@@ -2,21 +2,21 @@
 
 initTargetDistroRid()
 {
-    source ${__ProjectDir}/init-distro-rid.sh
+    source "${__ProjectDir}/init-distro-rid.sh"
 
     local passedRootfsDir=""
 
     # Only pass ROOTFS_DIR if cross is specified.
     if [ "$__CrossBuild" = 1 ]; then
-        passedRootfsDir=${ROOTFS_DIR}
+        passedRootfsDir="${ROOTFS_DIR}"
     fi
 
     initDistroRidGlobal "$__BuildOS" "$__BuildArch" "$__PortableBuild" "$passedRootfsDir"
 }
 
 isMSBuildOnNETCoreSupported()
 {
-    __isMSBuildOnNETCoreSupported=$__msbuildonunsupportedplatform
+    __isMSBuildOnNETCoreSupported="$__msbuildonunsupportedplatform"
```

Comment:
I wonder why you have added double quotes at many places even though they are not things that can contain spaces.

Reply:
The syntax highlighter in some editors and shellcheck linter highlights these variables and suggest to quote variables at all usage sites. We were inconsistent in quoting variable at usages, i have made it consistent in this file.

Reply:
@janvorli, can you take a look at changes?